website/docs/docs/deploy/job-settings.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Job settings"
+description: "Learn how to create and schedule jobs in dbt Cloud for the scheduler to run. Jobs help you build observability into transformation workflows with the in-app scheduling, logging, and alerting." 
+tags: [scheduler]
+---
+
+Jobs make it easy to run dbt commands against a project in the your cloud data platform, triggered either by schedule or events. Each job run in dbt Cloud will have a run history, run status and a run overview, which provides you with:
+
+- Job trigger type
+- Commit SHA
+- Environment name
+- Sources and documentation info 
+- Job run details, including run timing, [model timing data](#model-timing), and [artifacts](/docs/deploy/artifacts)
+- Detailed run steps with logs and their statuses
+
+You can create a job and configure it to run on [scheduled days and times](/docs/deploy/job-triggers#schedule-days) or enter a [custom cron schedule](/docs/deploy/job-triggers#custom-cron-schedules). 
+
+## Prerequisites
+
+- You must have a dbt Cloud account and [Developer seat license](/docs/cloud/manage-access/seats-and-users). If you don't, you can [sign up](https://www.getdbt.com/signup/) for a [free account](https://www.getdbt.com/pricing/). 
+- You must have a dbt project connected to a [data platform](/docs/cloud/connect-data-platform/about-connections).
+- You must [create and schedule a dbt Cloud job](#create-and-schedule-jobs).
+- You must have [access permission](/docs/cloud/manage-access/about-user-access) to read, create, edit, or run jobs.
+- You must set up a [deployment environment](/docs/collaborate/environments/dbt-cloud-environments). 
+
+## Create and schedule jobs {#create-and-schedule-jobs}
+
+1. Create a new job by clicking **Deploy** in the header, click **Jobs** and then **Create job**.
+1. Provide a job name, for example "Hourly Customer Job". 
+1. Under **Environment**, add the following:
+    * **Environment** &mdash; Link to an existing deployment environment.
+    * **dbt Version** &mdash; Select the dbt [version](/docs/dbt-versions/core). dbt Labs recommends inheriting the version from the environment settings.
+    * **Target Name** &mdash; Define the [target name](/docs/build/custom-target-names) for any dbt cloud job to correspond to settings in your project.
+    * **Threads** &mdash; The default value is 4 [threads](/docs/core/connect-data-platform/connection-profiles#understanding-threads). Increase the thread count to increase model execution concurrency. 
```

Comment:
Do you know if we need to add any caveats? "Creating too many threads will impact performance" or something like that?

Reply:
thanks for flagging, @matthewshaver ! 

checked with the PM. high thread count is fine